internal/db/storage/blob/s3/option.go

```diff
@@ -59,3 +59,15 @@ func WithMaxPartSize(size int64) Option {
 		}
 	}
 }
+
+func WithMaxPartSizeKB(kb int) Option {
```

Comment:
in this section, I think it's better to define measurement const like below
```
const (
    _ = iota
    KB int = 1 << (10 * iota)
    MB
    GB
    TB
)
```

Reply:
also, it's better to define by the string (10kb, 4mb, 5gb, etc...) and parse it in the option func.

Reply:
agree. i'll do it later.